mojaloop-payment-manager/values.yaml

```diff
@@ -424,7 +430,7 @@ scheme-adapter:
     image:
       registry: docker.io
       repository: mojaloop/sdk-scheme-adapter
-      tag: v23.5.1
+      tag: v23.6.0-snapshot.11
```

Comment:
Shall we release sdk-scheme-adapter first?

Reply:
I think this change is reverted.